cirq-core/cirq/transformers/align.py

```diff
@@ -73,4 +73,11 @@ def align_right(
     """
     if context is not None and context.deep is True:
         context = dataclasses.replace(context, deep=False)
-    return align_left(circuit[::-1], context=context)[::-1]
+    backwards = []
```

Comment:
Can we add a line comment (or to the doc string) to explain the logic here (similar to what you put in the PR description)?  Aligning things backwards is kind of confusing to think about.